eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -14,7 +14,7 @@ jobs:
 
       - script: |
           npm install -g npm@6.9.0
-        displayName: 'Install npm version 6.9.0'
+        displayName: "Install npm version 6.9.0"
```

Comment:
Why the change from single to double quotes? Our prettier config has singlequote https://github.com/Azure/azure-sdk-for-js/blob/master/.azure-pipelines/.prettierrc.yml, however it looks like we haven't moved that to our eng directory.

Reply:
I felt it was better to make one thing consistent than re-format the entire file.

Reply:
lol.. it looks like this file is the only one of the yml files that currently uses double quotes. @mitchdenny I know you were planning on taking a pass and adding the prettierrc.yml file under eng directory and when doing that it would be good to format them consistently.